src/gluon/events/project/TeamAssociated.ts

```diff
@@ -31,16 +33,21 @@ subscription TeamsLinkedToProjectEvent {
   }
 }
 `)
-export class TeamsLinkedToProject implements HandleEvent<any> {
+export class TeamsLinkedToProject extends BaseQMEvent implements HandleEvent<any> {
 
     public async handle(event: EventFired<any>, ctx: HandlerContext): Promise<HandlerResult> {
         logger.info(`Ingested TeamAssociated event: ${JSON.stringify(event.data)}`);
 
-        const teamsLinkedToProjectEvent = event.data.TeamsLinkedToProjectEvent[0];
+        try {
+            const teamsLinkedToProjectEvent = event.data.TeamsLinkedToProjectEvent[0];
 
-        const destination = await addressSlackChannelsFromContext(ctx, teamsLinkedToProjectEvent.team[0].slackIdentity.teamChannel);
-        return ctx.messageClient.send(`Your team has been successfully associated with ${teamsLinkedToProjectEvent.id}`,
-            destination);
+            const destination =  await addressSlackChannelsFromContext(ctx, teamsLinkedToProjectEvent.team[0].slackIdentity.teamChannel);
```

Comment:
Another extra space for some reason.
```suggestion
            const destination = await addressSlackChannelsFromContext(ctx, teamsLinkedToProjectEvent.team[0].slackIdentity.teamChannel);
```